diff -cr original/boost/move/move.hpp modified/boost/move/move.hpp
*** original/boost/move/move.hpp	Sun Oct 16 20:47:29 2011
--- modified/boost/move/move.hpp	Thu Jan  5 11:53:54 2012
***************
*** 25,30 ****
--- 25,32 ----
     #ifndef _CRT_SECURE_NO_DEPRECATE
     #define  BOOST_INTERPROCESS_CRT_SECURE_NO_DEPRECATE
     #define _CRT_SECURE_NO_DEPRECATE
+    #endif
+    #ifndef _SCL_SECURE_NO_WARNINGS
     #define _SCL_SECURE_NO_WARNINGS
     #endif
     #pragma warning (push)
