diff -cr original/Alc/ALc.c modified/Alc/ALc.c
*** original/Alc/ALc.c	Wed Feb  9 19:52:20 2011
--- modified/Alc/ALc.c	Sat Aug  6 15:28:10 2011
***************
*** 337,342 ****
--- 337,344 ----
  ///////////////////////////////////////////////////////
  // Global Variables
  
+ static ALCuint g_ulInitCount = 0;
+ 
  static ALCdevice *g_pDeviceList = NULL;
  static ALCuint    g_ulDeviceCount = 0;
  
***************
*** 2334,2337 ****
--- 2336,2355 ----
      }
  }
  
+ ALC_API void ALC_APIENTRY alcInit(void)
+ {
+ 	if (g_ulInitCount == 0)
+ 		DllMain(NULL, DLL_PROCESS_ATTACH, NULL);
+ 
+ 	g_ulInitCount++;
+ }
+ 
+ ALC_API void ALC_APIENTRY alcRelease(void)
+ {
+ 	g_ulInitCount--;
+ 
+ 	if (g_ulInitCount == 0)
+ 		DllMain(NULL, DLL_PROCESS_DETACH, NULL);
+ }
+ 
  ///////////////////////////////////////////////////////
diff -cr original/include/AL/alc.h modified/include/AL/alc.h
*** original/include/AL/alc.h	Sun Nov 28 22:51:16 2010
--- modified/include/AL/alc.h	Sat Aug  6 15:37:47 2011
***************
*** 177,182 ****
--- 177,190 ----
  
  
  /*
+  * Initialization and deinitialization
+  */
+ ALC_API void            ALC_APIENTRY alcInit( void );
+ 
+ ALC_API void            ALC_APIENTRY alcRelease( void );
+ 
+ 
+ /*
   * Context Management
   */
  ALC_API ALCcontext *    ALC_APIENTRY alcCreateContext( ALCdevice *device, const ALCint* attrlist );
diff -cr original/utils/openal-info.c modified/utils/openal-info.c
*** original/utils/openal-info.c	Wed Aug  4 12:41:12 2010
--- modified/utils/openal-info.c	Sat Aug  6 14:59:10 2011
***************
*** 279,284 ****
--- 279,286 ----
      ALCdevice *device;
      ALCcontext *context;
  
+ 	alcInit();
+ 
      if(argc > 1 && (strcmp(argv[1], "--help") == 0 ||
                      strcmp(argv[1], "-h") == 0))
      {
***************
*** 328,332 ****
--- 330,336 ----
      alcDestroyContext(context);
      alcCloseDevice(device);
  
+ 	alcRelease();
+ 
      return 0;
  }
