diff -cr original/modules/highgui/CMakeLists.txt modified/modules/highgui/CMakeLists.txt
*** original/modules/highgui/CMakeLists.txt	Mon Oct 17 00:56:32 2011
--- modified/modules/highgui/CMakeLists.txt	Mon Oct 17 04:04:29 2011
***************
*** 68,77 ****
      set(GRFMT_LIBS ${GRFMT_LIBS} ${OPENEXR_LIBRARIES})
  endif()
  
- if(MSVC)
-     set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} /NODEFAULTLIB:libcmt.lib")
- endif()
- 
  file(GLOB grfmt_hdrs src/grfmt*.hpp)
  file(GLOB grfmt_srcs src/grfmt*.cpp)
  set(grfmt_hdrs src/bitstrm.hpp ${grfmt_hdrs})
--- 68,73 ----
